fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/controlTokens/TooltipTokens.kt

```diff
@@ -0,0 +1,83 @@
+package com.microsoft.fluentui.theme.token.controlTokens
+
+import android.os.Parcelable
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.microsoft.fluentui.theme.FluentTheme
+import com.microsoft.fluentui.theme.token.ControlInfo
+import com.microsoft.fluentui.theme.token.FluentAliasTokens
+import com.microsoft.fluentui.theme.token.FluentGlobalTokens
+import com.microsoft.fluentui.theme.token.IControlToken
+import kotlinx.parcelize.Parcelize
+
+class TooltipInfo : ControlInfo
+
+@Parcelize
+open class TooltipTokens : IControlToken, Parcelable {
+    @Composable
+    open fun backgroundBrush(tooltipInfo: TooltipInfo): Brush =
+        SolidColor(
+            FluentTheme.aliasTokens.neutralBackgroundColor[FluentAliasTokens.NeutralBackgroundColorTokens.BackgroundDarkStatic].value(
+                themeMode = FluentTheme.themeMode
+            )
+        )
+
+    @Composable
+    open fun tipColor(tooltipInfo: TooltipInfo): Color =
```

Comment:
can we make tip a brush just like background brush

Reply:
Icon & Image does not take brush for tint param